src/linuxforhealth/csvtofhir/pipeline/tasks.py

```diff
@@ -196,6 +205,17 @@ def format_date(
     """
     Formats date string values within a DataFrame column to a target date format.
 
+    Example:
```

Comment:
If we ever publish the "API" separately or autogen documentation, it may be worth to look at Sphinx.

https://chiplicity.readthedocs.io/en/latest/Using_Sphinx/ShowingCodeExamplesInSphinx.html